javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+
+const number = document.querySelector('.countdown-display');
+
+let counter = 4;
+function countdown() {
+  const boom = number.textContent = 'boom';
```

Comment:
I would move this to inside the `if` statement. We only want this to happen when the number hits 0